include/librealsense2/dds/topics/notification/notification-msg.h

```diff
@@ -87,6 +88,21 @@ class notification
         motion_stream_profile profiles[MAX_MOTION_PROFILES];
     };
 
+    enum class control_result : uint16_t
+    {
+        SUCCESSFUL,
+        FAILED,
```

Comment:
We need to define a better error format for a response (todo)

Reply:
There is a TODO comment there